packages/frontend/src/components/DashboardFilter/ActiveFilters/index.tsx

```diff
@@ -37,6 +45,45 @@ const ActiveFilters: FC<ActiveFiltersProps> = ({
         (c) => c.updateDimensionDashboardFilter,
     );
 
+    const makeTabGetter = useCallback(
```

Comment:
> if the filter does not apply to any Tiles on the current Tab, show the filter as inactive

The initial ticket doesn't mention this behaviour. Was this brought up by someone?
Even though this could be a clever way to identify which filters are being used in the tab, as a user, I was confused. 
The filter rules were disabled and said they were only applied to the first tab but when I went to edit them, it was marked as enabled.

<img width="902" alt="Screenshot 2024-10-21 at 14 37 45" src="https://github.com/user-attachments/assets/995d4255-5965-4971-b002-677450662646">

Reply:
The issue mentions "disabled" rather than "inactive" but I think I see the main issue you're mentioning. This PR currently checks if each filter is being applied to any tile on the current tab. I think you're suggesting that it should check if each filter is enabled for a tab, even if there are no tiles on the current tab. That makes sense. I'll take a look.